pkg/uwb-core/contrib/uwb_core_init.c

```diff
@@ -0,0 +1,79 @@
+/*
+ * Copyright (C) 2020 Inria
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @ingroup     pkg_uwb
+ * @{
+ *
+ * @file
+ * @brief       uwb-core bootstraping core
```

Comment:
```suggestion
 * @brief       uwb-core bootstrapping core
```

Please check if there are others ;)